scripts/make-packages.sh

```diff
@@ -183,7 +184,14 @@ if [ "$OS" == "linux" ]; then
      FILES="${FILES} ${ARTIFACTS}/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}=/usr/${LIBDIR}/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}"
   fi
 
-  if [ "$UBUNTU" == '2404' ]; then
+  # XDP means Ubuntu 24.04 which installs libssl3t64 instead of libssl3
```

Comment:
I would call this time64Distro or something like that actually, because there might be some cases in the future where you might not want to use XDP and get rid of it as dependency on different distros but might still require you to get this t64 version if libssl3